src/main/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/service/LicenceService.kt

```
@@ -1063,32 +1069,38 @@ class LicenceService(
   }
 
   @Transactional
-  fun timeout(licence: CrdLicence, reason: String? = null) {
-    val timedOutLicence = licence.timeOut()
-    licenceRepository.saveAndFlush(timedOutLicence)
+  fun timeout(licence: EntityLicence, reason: String? = null) {
+    if (licence !is SupportsHardStop) {
+      log.warn("can only timeout licences that support hard stop")
```

should never happen, maybe should throw an exception?